src/test/java/training/sign_up/sign_up.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Register a new user
+
+  Background:
+    * url someUrl
+
+  Scenario: Register successfully
```

Comment:
I recommend you use a Outline Scenario for this feature to validate all the tests cases :).